manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/pipeline.yaml

```diff
@@ -666,7 +681,8 @@ spec:
                   name: mysql-credential
                   key: password
             {{ end }}
-          image: {{ .Values.images.apiserver }}
+          # image: {{ .Values.images.apiserver }}
+          image: gcr.io/ml-pipeline/google/pipelines:jxzheng-test
```

Comment:
Just out of curiosity: Do our tests fail to replace the container image name with the newly build test image?

Reply:
Sorry, need to clean this up :) Thanks for catching!